### ISSUE TITLE
Question about LLM

### ISSUE THREAD

**Issue by upq** [m87-labs/moondream#73]
Hello, great work first of all, secondly i would like to know if it is possible to use a large language model like llama inside moondream to make it able to write promotional material from images, such as product descriptions, social media posts, sales copy etc..

Thank you so much for your efforts and have a great one.

**Comment by BindingOx** [comment]
yes, you could use an lm studio to do so like using it to collect image descriptions and write summaries based on them. check out "100% Local Tiny AI Vision Language Model (1.6B) - Very Impressive!!" on youtube.
